README.md

```diff
@@ -500,6 +500,32 @@ server thus resetting the storage completely.
 Note that if many hundreds of megabytes of content (queue message or blob content) are stored in-memory, it can take
 noticeably longer than usual for the process to terminate since all the consumed memory needs to be released.
 
+### Use Blob Versioning
+
+#### How it works
+
+Blob Versioning was implemented to follow the exact guidelines outlined [here](https://learn.microsoft.com/en-us/azure/storage/blobs/versioning-overview), excluding interactions with soft delete, blob expiration, SAS URIs, since Azurite does not support that.
+
+#### How to use it
+
+Optional. By default, this is disabled. To enable it, there are two CLI args you can use: accountConfigFilePath, accountConfigAsJson.
+
+Blob versioning is enabled by leveraging the [AccountModel](src/blob/AccountModel.ts). The account model is an abstraction to configure the storage account. Currently, it only supports configuring blob versioning.
+
+accountConfigFilePath lets you pass in the path to a json file modeled after the AccountModel, which is then used to configure Azurite.
+
+```bash
+azurite --accountConfigFilePath "./myAccountModel.json"
+```
+
+accountConfigAsJson allows you to pass a json string as a CLI arg to configure the account as well.
+
+```bash
+azurite --accountConfigAsJson "{ \"isBlobVersioningEnabled\": true }"
```

Comment:
Please add the 2 new parameters for all place for all parameters in different executable. 
And the long description can be added in place like: https://github.com/Azure/Azurite?tab=readme-ov-file#use-in-memory-storage